pkg/virt-handler/vm.go

```diff
@@ -2663,6 +2664,11 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 			return fmt.Errorf("failed to adjust resources: %v", err)
 		}
 	} else if vmi.IsRunning() {
+		if err := d.hotPlugSriovInterfaces(vmi); err != nil {
```

Comment:
Are we reaching this point on a target migration flow?
I'm asking, because if we reach it too soon, we may have a problem. If we reach it too late, we increase the time the target is not fully functional.

Reply:
This part is out of the migration flow, I will verify that there is no issue with doing the hot-plug here